<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SariSariSleuth</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 71, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 216, 216);
border-radius: 30px;</string>
      </property>
      <widget class="QWidget" name="container_buttons" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>781</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>561</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="main_buttons">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sales_btn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb();</string>
           </property>
           <property name="text">
            <string>Sales</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="transt_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb();</string>
           </property>
           <property name="text">
            <string>Transactions</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logging_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb();</string>
           </property>
           <property name="text">
            <string>Stck lging</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="misctab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb();</string>
           </property>
           <property name="text">
            <string>Tab 4</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>761</width>
         <height>471</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="tabs">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>451</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab4"/>
        <widget class="QWidget" name="sales">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>60</y>
            <width>371</width>
            <height>191</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
border-color: rgb(255, 255, 255);
background-image: url(:/resources/sold.png);
border-radius: 15px;
</string>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>371</width>
             <height>191</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 900 9pt &quot;Montserrat&quot;;
font-size: 100px;
color: rgb(0, 71, 255);
qproperty-alignment: 'AlignCenter';</string>
           </property>
           <property name="text">
            <string>01</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>120</y>
            <width>371</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 71, 255);
border-radius: 15px;
color: rgb(0, 71, 255);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="openCameraButton">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>320</y>
            <width>150</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
font: 700 9pt &quot;Montserrat&quot;;
color: rgb(0, 71, 255);
border-radius: 10px;
</string>
          </property>
          <property name="text">
           <string>Open Camera</string>
          </property>
         </widget>
         <widget class="QPushButton" name="manualAddButton">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>320</y>
            <width>161</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
font: 700 9pt &quot;Montserrat&quot;;
color: rgb(0, 71, 255);
border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>Manual Add</string>
          </property>
         </widget>
         <zorder>widget_2</zorder>
         <zorder>widget_3</zorder>
         <zorder>openCameraButton</zorder>
         <zorder>manualAddButton</zorder>
        </widget>
        <widget class="QWidget" name="stocklogging">
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>721</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableView" name="transactionTableView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>328</width>
                 <height>52</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="confirmButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(38, 255, 0);
font: 700 9pt &quot;Montserrat&quot;;
color: rgb(36, 84, 28);
border: 20px solid transparent;
border-radius: 15px;</string>
               </property>
               <property name="text">
                <string>Confirm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteTransactionButton">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 0);
font: 700 9pt &quot;Montserrat&quot;;
color: rgb(255, 255, 255);
border: 20px solid transparent;
border-radius: 15px;</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="transactions">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>681</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Filter:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filterLineEdit">
               <property name="placeholderText">
                <string>Search products...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="stockTableView">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editButton">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
